Firestore/core/src/firebase/firestore/local/leveldb_key.h

```diff
@@ -416,6 +416,18 @@ class LevelDbDocumentTargetKey {
    */
   static std::string SentinelKey(const model::DocumentKey& document_key);
 
+  /**
+   * Given a sequence number, encodes it for storage in a sentinel row.
+   */
+  static std::string EncodeSentinel(
```

Comment:
I'm not opposed to putting this here, but the name is slightly confusing when mixed in with e.g. `SentinelKey`. Maybe `EncodeSentinelValue` might help?

Alternatively, you could make these members of the LocalSerializer, which is otherwise concerned with reading and writing values, but it's not a great fit there either because mostly it's concerned with translating between model objects and their equivalent protocol buffer messages.

Another possible home is just as static members of the LevelDbQueryCache, which is otherwise where these things are used. It's not unprecedented to have migrations share code that way.

Reply:
Left it in place, but changed to `EncodeSentinelValue`/`DecodeSentinelValue`.